app/portainer/components/datatables/components/ActionsMenu.tsx

```diff
@@ -0,0 +1,28 @@
+import {Menu, MenuList, MenuButton} from "@reach/menu-button";
+import clsx from "clsx";
+import {ReactNode} from "react";
```

Comment:
Inconsistent spacing around imports.   Now you're not using { } for single entries and and other times you are?

Reply:
It's expected that some imports use brackets and others don't, that is something that depends on how the imported module is declared, and wether an entire module is imported (for example `clsx` here) or only specific things from a module (like 3 components from `@reach/menu-button` here).

I'm not an experte but there is more info here: https://stackoverflow.com/questions/36795819/when-should-i-use-curly-braces-for-es6-import